_data/social.yml

```diff
@@ -150,6 +150,12 @@ websites:
       exceptions:
           text: "SMS-capable phone required."
       doc: https://en.support.wordpress.com/security/two-step-authentication/
+      
+    - name: Xing.com
```

Comment:
The seem to use the name XING themselves. Maybe change the name to their correct name?